mitodl/micromasters
# recent posts component

#### What are the relevant tickets?

closes #3476 

#### What's this PR do?

This improves the existing 'discussions' card to implement the mocks, by hitting the OD backend to fetch the frontpage and then rendering it.

#### How should this be manually tested?

Make sure you have all the stuff set up for OD and MM to talk to each other. You should also have `FEATURE_OPEN_DISCUSSIONS_POST_UI` set to `True`. Then you should see something like this on the dashboard:

![obama](https://user-images.githubusercontent.com/6207644/30759654-04e82768-9fa5-11e7-8f12-ea7b24359caa.png)

If you don't set the `POST_UI` feature flag or you don't set the redirect URL env var you shouldn't see the card. If you do, you should!
